cypress/integration/users/create-users.spec.js

```diff
@@ -0,0 +1,21 @@
+describe('Creating user', () => {
+  beforeEach(() => {
+    cy.login('diku_admin', 'admin');
+  });
+
+  it('should be possible', function () {
+    Cypress.on('uncaught:exception', () => {
+      return false;
+    });
+
+    cy.visit('/users/create');
+    cy.get('#adduser_lastname').type('Test123');
+    cy.get('#adduser_group').select('undergrad (Undergraduate Student)', { force: true });
+    cy.get('#adduser_email').type('test@folio.org', { force: true });
+    cy.get('#clickable-save').click({ force: true });
+    cy.get('#pane-userdetails:contains(Test123)');
+    cy.get('#pane-userdetails [data-pane-header-actions-dropdown="true"]').click();
```

Comment:
if you want to clean up data, use API for this, your test case is about user creation, you shouldn't click delete buttons in it

Reply:
explained offline